src/reverb/doc/cron-for-ovh/cron-sync-orders.php

```diff
@@ -0,0 +1,16 @@
+<?php
+// Get cURL resource
+$curl = curl_init();
+// Set some options - we are passing in a useragent too here
+curl_setopt_array(
+    $curl,
+    array(
+        CURLOPT_RETURNTRANSFER => 1,
+        CURLOPT_URL => '[ADD YOUR DOMAIN URL]/modules/reverb/cron.php?code=orders',
```

Comment:
This release is principaly a fix about the pagination of the Category mapping.
The 3 files about OVH, we are writting a description in French/English but it will be available to the next release with the new feature "mass edit". Aabesh is aware.
thank you.